supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -0,0 +1,383 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Product data information for supporters."
+
+Parameters:
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - PROD
+      - UAT
+      - CODE
+    Default: CODE
+
+Conditions:
+  CreateProdResources: !Equals [!Ref "Stage", "PROD"]
+
+Mappings:
+  StageVariables:
+    CODE:
+      S3Bucket: supporter-product-data-export-code
+      # Every 10 mins between 7am and 7pm Mon-Fri
+      scheduleRate: "cron(*/10 7-19 ? * 1-5 *)"
+    UAT:
```

Comment:
so we have a UAT stage in AWS - normally stages in AWS are code/prod and touchpoint stages are dev/uat/prod.  Why that way?

Reply:
Just for simplicity really, the 2 stages and 3 environments configuration gets really complicated and I think it's overkill for this

Reply:
in that case what is CODE?  I could understand dev/uat/prod or dev/code/prod but code/uat/prod seems confusing

Reply:
Let's discuss

Reply:
Updated as discussed